packages/offix-client/src/OfflineClient.ts

```diff
@@ -60,13 +61,12 @@ export class OfflineClient implements ListenerProvider {
   /**
   * Initialize client
   */
-  public async init(): Promise<ApolloOfflineClient> {
+  public async init(httpOptions?: HttpLink.Options): Promise<ApolloOfflineClient> {
     await this.store.init();
     const cache = await buildCachePersistence(this.config.cacheStorage);
     const offlineLink = await createOfflineLink(this.config, this.store);
     const conflictLink = await createConflictLink(this.config);
-    const link = await createDefaultLink(this.config, offlineLink, conflictLink, cache);
-
+    const link = await createDefaultLink(this.config, offlineLink, conflictLink, cache, httpOptions);
```

Comment:
This is optional in your `init` method so you will need to check it exists before passing it here or make it optional on the methods that use it.